<launch>

  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>
  <arg     if="$(arg debug)" name="launch_prefix" value="debug_roslaunch"/>

  <node pkg="balloon_filter" type="conic_rheiv" name="tests" output="screen" launch-prefix="$(arg launch_prefix)" />

</launch>


